pkg/project/auth/cache.go

```diff
@@ -86,6 +116,7 @@ type AuthorizationCache struct {
 	policyBindingsNamespacer client.PolicyBindingsNamespacer
 	policiesNamespacer       client.PoliciesNamespacer
 
+	state       State
```

Comment:
Better name.  I can't tell what this is for without digging through the type, it's implementations, and knowledge of what the reflector is doing.